src/Query/Builder.php

```diff
@@ -1247,10 +1304,16 @@ protected function compileWhereTime(array $where): array
     {
         extract($where);
 
-        $where['operator'] = $operator;
-        $where['value'] = $value;
-
-        return $this->compileWhereBasic($where);
+        return [
```

Comment:
I noted the original author's comment in https://github.com/jenssegers/laravel-mongodb/pull/2376:

> whereTime isnt possible to fix rn since i cant find logic to query like time() on sql

Does this address that concern?

Reply:
Yes, fixed. There is no dedicated operator, but this was possible using `$dateToString`.